python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -166,6 +166,7 @@ class ComputeExposureSummaryStatsTask(pipeBase.Task):
     - psfIxy
     - ra
     - dec
+    - pixelScale (arcsec/pixel)
```

Comment:
Should expTime also be included here? I know it's not really "calculated", but I feel that this list should say what's included in the summary, not just what's "calculated".